action/protocol/staking/candidate_statemanager.go

```diff
@@ -56,7 +56,7 @@ type (
 		Upsert(*Candidate) error
 		CreditBucketPool(*big.Int) error
 		DebitBucketPool(*big.Int, bool) error
-		Commit() error
+		Commit(bool) error
```

Comment:
this fix should only happen one-time at the HF height, why need to add it as an extra input